openshift/fluentd.dc.yaml

```diff
@@ -0,0 +1,308 @@
+---
+apiVersion: v1
+kind: Template
+labels:
+  app: ${APP_NAME}-${INSTANCE}
+  env: ${LABEL_ENV}
```

Comment:
We don't "really" need this -> this label is relatively superfluous as you'd only ever have one fluentd instance per openshift namespace; whether it be dev, test or prod, the namespace itself already defines what the environment is. You can feasibly drop all `env: ${LABEL_ENV}` references from this template.